### PR TITLE
[DOCFIX] Update scalability tuning documentation

### PR DESCRIPTION
**Pull request by ZacBlanco**

### CODE REVIEW

File: docs/en/operation/Scalability-Tuning.md [hunk 1]
```diff
@@ -73,10 +75,10 @@ ALLUXIO_WORKER_JAVA_OPTS+=" -Xms4g -Xmx4g"
 The frequency in which a worker checks in with the master is set by the following properties:
```
Comment (madanadit) [inline docs/en/operation/Scalability-Tuning.md]: nit: not related to this PR, but it should be "The frequency _with_ which" and "properties" to "property"

File: docs/en/operation/Scalability-Tuning.md [hunk 2]
```diff
@@ -73,10 +75,10 @@ ALLUXIO_WORKER_JAVA_OPTS+=" -Xms4g -Xmx4g"
...
 ```properties
 alluxio.worker.block.heartbeat.interval=1s
-alluxio.worker.filesystem.heartbeat.interval=1s
 ```
-`alluxio.worker.block.heartbeat.interval` controls the heartbeat intervals for the block service in Alluxio and
-`alluxio.worker.filesystem.heartbeat.interval` for the filesystem service.
+
+`alluxio.worker.block.heartbeat.interval` controls the heartbeat intervals for the block service in
```
Comment (madanadit) [inline docs/en/operation/Scalability-Tuning.md]: "intervals" to "Interval"?

File: docs/en/operation/Scalability-Tuning.md (hunk 1 above)
Comment (ZacBlanco) [reply docs/en/operation/Scalability-Tuning.md]: done

File: docs/en/operation/Scalability-Tuning.md (hunk 2 above)
Comment (ZacBlanco) [reply docs/en/operation/Scalability-Tuning.md]: updated

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please.

**Pull request merged**